bigml/association.php

```diff
@@ -91,11 +91,11 @@ public function __construct($association, $api=null) {
          if (file_exists($association))
          {
             $association = json_decode(file_get_contents($association));
-         } else if (!($api::_checkAssociationId($association)) ) {
+         } else if (!($api->_checkAssociationId($association)) ) {
             error_log("Wrong association id");
             return null;
          } else {
-            $association = $api::retrieve_resource($association, $api::ONLY_MODEL);
+            $association = $api::retrieve_resource($association, BigML::ONLY_MODEL);
```

Comment:
$api->retrieve_resource maybe?

Reply:
Good catch, though for the beauty that PHP is, $api:: and $api-> are the same (when $api is a reference to an object)... the backward compatibility problem only arises for uses like BigML::